resources/views/helper.php

```diff
@@ -1,7 +1,7 @@
 <?= '<?php' ?>
 <?php
 /**
- * @var \Barryvdh\LaravelIdeHelper\Alias[][] $namespaces_by_alias_ns
+ * @var \Barryvdh\LaravelIdeHelper\Alias[][]
```

Comment:
That seems… unfortunate? No idea why it's removed 💥 

Doesn't affect code but IDE hinting within the helper isn't so nice for this varaible. Maybe we can leave with that for now and try to fix it (or report a php-cs-fixer bug) later 🤷‍♀️